node/libs/protobuf/src/repr.rs

```diff
@@ -0,0 +1,20 @@
+//! `ProtoRepr` utilities.
```

Comment:
"Trait for defining proto conversion for external types."

Reply:
Fixed.